framework/src/util/PropsCheck.h

```diff
@@ -0,0 +1,67 @@
+/*=============================================================================
```

Comment:
Our last SW at Google book club discussed documentation. Maybe with brand new files we could consider increasing our documentation? Some comments to consider, file comment, class comments and function comments.

Reply:
Added.